input/kafkamdm/kafkamdm.go

```diff
@@ -256,13 +260,11 @@ func (k *KafkaMdm) tryGetOffset(topic string, partition int32, offset int64, att
 	return val, err
 }
 
-// this will continually consume from the topic until k.stopConsuming is triggered.
+// this will continually consume from the topic until k.shutdown is triggered.
```

Comment:
shouldn't the function comments start with the function name?
something like 

```
consumePartition will continously ... etc
```

Reply:
yep